lib/src/realm_class.dart

```diff
@@ -31,7 +31,7 @@ export 'results.dart' hide RealmResultsInternal;
 export 'realm_object.dart'
     hide RealmObjectInternal, RealmAccessor, RealmValuesAccessor, RealmMetadata, RealmCoreAccessor, RealmClassMetadata, RealmPropertyMetadata;
 export "configuration.dart" hide ConfigurationInternal;
-export 'package:realm_common/realm_common.dart' show RealmModel, PrimaryKey, Ignored, MapTo, Indexed, RealmPropertyType, RealmCollectionType;
+export 'package:realm_common/realm_common.dart' hide Decimal128, RealmAny, Uuid;
```

Comment:
I liked the explicit show more. Thus we have control what is exposed and we don't slip something unintentionally

Reply:
yes indeed its long but it could be single Error type (per my comment above) and we better have a long list here than unintentionally exposing something. I agree it's not ideal, but since there are no visibility modifiers...

Reply:
To me it is clearer to read that we explicitly would like to hide 3 classes. It would easy to hide one of these in a long list of show declarations.